contrail_api_cli_extra/clean/refs.py

```diff
@@ -69,10 +71,22 @@ def _remove_refs(self, paths):
         printo("[%s -> %s] deleted" % (source, target))
         self._remove_refs(paths[2:])
 
-    def __call__(self, paths=None, ref_type=None, target_type=None, cassandra_servers=None, **kwargs):
+    def _read_file(self, resources_file):
+        paths = []
+        with open(resources_file, 'r') as f:
+            for l in f:
+                paths.append(l.split())
+        return paths
+
+    def __call__(self, paths=None, resources_file=None, ref_type=None, target_type=None, cassandra_servers=None, **kwargs):
         super(CleanRefs, self).__call__(**kwargs)
         pool = ConnectionPool('config_db_uuid', server_list=cassandra_servers)
         self.uuid_cf = ColumnFamily(pool, 'obj_uuid_table')
         self.ref_type = ref_type
         self.target_type = target_type
+        if paths is not None and resources_file is not None :
+            exit("You must specify resource ids as arguments OR (exclusive) as file")
```

Comment:
It would be better to be able to expose this constraint in the help. 
Maybe a workaround would be to append paths from the file to paths provided on the CLI, instead of using a exclusive or.

Reply:
Ok no problem with that. :+1: 
I found the first way more clean :)